### DISCUSSION TITLE
Let us delete projects

### DISCUSSION THREAD

**Discussion by openxthinking** [community/community#14121] [Projects and Issues] [answered]
It'd be cool to be able to delete projects not just archive them.

**Comment by 2percentsilk** [comment, marked as answer]
We have recently shipped the ability to [delete projects](https://github.blog/changelog/2022-05-19-the-new-github-issues-may-19th-update/#%E2%9C%82%EF%B8%8F-delete-projects-you-no-longer-need). Give it a try and let us know how it goes.
